packages/cli/test/unit/repositories/execution.repository.test.ts

```diff
@@ -0,0 +1,35 @@
+import { mock } from 'jest-mock-extended';
+import Container from 'typedi';
+import { EntityManager, DataSource } from 'typeorm';
+import { mockInstance } from '../../shared/mocking';
+import { ExecutionRepository } from '@/databases/repositories/execution.repository';
+
+describe('ExecutionRepository', () => {
+	const entityManager = mockInstance(EntityManager);
+	const dataSource = mockInstance(DataSource, { manager: entityManager });
+	dataSource.getMetadata.mockReturnValue(mock());
+	Object.assign(entityManager, { connection: dataSource });
+
+	const executionRepository = Container.get(ExecutionRepository);
+
+	describe('getWaitingExecutions()', () => {
+		test('should', async () => {
```

Comment:
can we also add tests to cover `dbType === 'sqlite'` and `dbType !== 'sqlite'` cases?
You could mock config using `jest.spyOn(config, 'getEnv')`